decidim-core/app/cells/decidim/redesigned_follow_button_cell.rb

```diff
@@ -9,10 +9,18 @@ def show
       render
     end
 
+    def followers_count
```

Comment:
Just a question here

why don't you use `model.followers.size`? and benefit from Rails's  internall counter caching mechanism?

Reply:
You are right, I copied this method from the old cell and didn't inspect it. Changed